src/lib/components/Flowbite/FlowbiteTheme.ts

```diff
@@ -136,6 +138,13 @@ export interface FlowbitePositions {
   'center-right': string;
 }
 
+export interface FlowbitePlacements {
+  left: { on: string; off: string };
```

Comment:
Isn't better to create an interface to `on: string; off: string;`?

Reply:
Ok